ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 set -o nounset
-set -o errexit
-set -o pipefail
```

Comment:
Did you want to keep these lines, and drop the `|| exit`, now that you have `errexit` handling wrapped more closely around the `install-status.txt` check?